gateway/dataimpl/server_v1/collectionadminserver.go

```diff
@@ -147,14 +163,28 @@ func (s *CollectionAdminServer) CreateCollection(
 	ctx context.Context,
 	in *admin_collection_v1.CreateCollectionRequest,
 ) (*admin_collection_v1.CreateCollectionResponse, error) {
+	apiVersion, err := apiversion.FromContext(ctx)
```

Comment:
Might be nice to have a helper function which takes a version and returns if the version is supported by the client. Not needed to for this PR though.